README.md

```diff
@@ -22,6 +22,14 @@ because otherwise Windmill tries to make the tests (as with ALL python files) in
 
 ### Development
 
+In Windmill, scripts and flows can be written in Python, TypeScript, Go, and a number of other languages.
+
+The `f/` directory is designated for storing code in a workspace, and will be used when synchronizing the contents of this repository with a server.
```

Comment:
I think what you have is fine. Just a note that (as of what's on `main` now) `wmill.yaml` will also sync a `u/` directory if it exists. This is meant for user-space scripts.  Maybe that doesn't make sense in this repo.

Reply:
I added a note about this.

Reply:
I agree with what you wrote.  I think to actually not use the `u/` convention, as you've added, you also need to remove `'u/**'`  from `wmill.yaml`'s `includes`:

    includes: ['f/**', 'u/**']